source/plugin/configuration/loaders.rst

```diff
@@ -140,22 +140,17 @@ Example: Loading a Default Config from the Plugin Jar File
 
 .. code-block:: java
 
-    import java.net.URL;
-
+    Sponge.getAssetManager().getAsset(myplugin, "defaultConfig.conf").get().copyToFile(path, false, true);
```

Comment:
Please use the same asset name everywhere. It does not matter which one you use.